components/project-loader/project-loader.jsx

```diff
@@ -32,20 +40,12 @@ export default React.createClass({
     this.setState(this.getInitialState(), () => {
       this.fetchData(this.props.bookmarkedOnly, nextProps);
     });
-  },
+  }
+
   createQueryParams(params) {
     let combinedParams = Object.assign({}, params);
 
     if (combinedParams.ids) {
-      // The `ids` query param is only used on the bookmarks(favs) page
```

Comment:
can we put this fairly large comment block back? If it requires an update, we should do that in a separate issue